apps/condo/domains/organization/utils/clientSchema/OrganizationEmployee.ts

```diff
@@ -53,39 +60,58 @@ function convertGQLItemToFormSelectState (item: OrganizationEmployee): { value:
 
     return { value: item.id, label: name }
 }
-
-function convertToGQLInput (state: IOrganizationEmployeeFormState): OrganizationEmployeeUpdateInput {
+type RelateToManyInput = {
+    connect?: { id: any }[],
+    disconnect?: { id: any }[],
+}
+function convertToGQLInput (state: IOrganizationEmployeeFormState, obj: IOrganizationEmployeeUIState): OrganizationEmployeeUpdateInput {
     const sender = getClientSideSenderInfo()
     const result = { dv: 1, sender }
-
-    // TODO(Dimitreee): refactor client utils to add disconectFeature, think about solution
-    for (const fieldName of Object.keys(state)) {
-        if (Object(state).hasOwnProperty(fieldName) && RELATIONS.includes(fieldName)) {
-            const fieldValue = get(state, fieldName)
-            const id = get(fieldValue, 'id')
-
-            if (id) {
-                result[fieldName] = {
-                    connect: {
-                        id,
-                    },
-                }
-            } else if (fieldValue) {
-                result[fieldName] = {
-                    connect: {
-                        id: state[fieldName],
-                    },
+    for (const attr of Object.keys(state)) {
```

Comment:
We need to reuse this logic somehow. It is copy-pasted from Division and looks like we will need to change this code (better if it will be in one place)

Reply:
yes

Reply:
I will make separate pr with refactoring